smartsim/_core/launcher/step/mpiStep.py

```diff
@@ -46,8 +45,20 @@ def __init__(self, name, cwd, run_settings):
         :param run_settings: run settings for entity
         :type run_settings: RunSettings
         """
+
         super().__init__(name, cwd)
+
+        self._supported_launchers = [
```

Comment:
Can this be a class var so the one list is shared across all instances of the class (assuming that this list is read only)?

Reply:
Good call